components/cli/pkg/commands/init.go

```diff
@@ -100,7 +104,17 @@ func RunInit() {
 		"# + iName - The Image name\n" +
 		"# + instances - The map dependency instances of the Cell instance to be created\n" +
 		"# + return - The Cell instance\n" +
-		"public function run(cellery:ImageName iName, map<string> instances) returns error? {\n" +
+		"public function run(cellery:ImageName iName, map<cellery:ImageName> instances) returns error? {\n" +
+		"    string VHostName = config:getAsString(\"VHOST_NAME\");\n" +
```

Comment:
Variable name should start with simple case, and use camel case for variable naming.